src/nvim/ui.coffee

```diff
@@ -23,6 +23,40 @@ get_vim_button_name = (button, e) ->
   kn += button + '>'
   kn
 
+# borrowed from underscore@acf826fa42ba6
+now = Date.now or ->
+  (new Date).getTime()
+debounce = (func, wait, immediate) ->
+  timeout = undefined
+  args = undefined
+  context = undefined
+  timestamp = undefined
+  result = undefined
+
+  later = ->
+    last = _.now() - timestamp
```

Comment:
You forgot to remove `_` here. In fact, just use `Date.now`, the `now` function is not necessary.